service/src/LevelZero.hpp

```diff
@@ -29,12 +29,71 @@ namespace geopm
             /// @brief Number of GPUs on the platform.
             /// @return Number of LevelZero GPUs.
             virtual int num_gpu() const = 0;
-
             /// @brief Number of GPUs on the platform.
             /// @param [in] domain The GEOPM domain type being targeted
             /// @return Number of LevelZero GPUs or GPU chips.
             virtual int num_gpu(int domain) const = 0;
-
+            /// @brief Get the number of LevelZero RAS domains of a certain type
```

Comment:
Please move all of these API additions to the bottom of the public section.  New APIs traditionally go at the bottom, not in the middle.

Reply:
Noted. Fixed.